debian/changelog

```diff
@@ -0,0 +1,5 @@
+authd (0.1) noble; urgency=medium
+
+  * Initial release
```

Comment:
Maybe it could be nice to also include the changes we did so far?

That would happen if you use:
 `gbp dch --multimaint-merge`

Although as said previously it would be something to be done at actual release time.